codecs/ffmpeg-binutils.patch

```diff
@@ -0,0 +1,72 @@
+From: Rémi Denis-Courmont <remi@remlab.net>
```

Comment:
That's fine. Can you please file an issue to update to a newer ffmpeg major version against:
https://github.com/flathub/org.gnome.Totem.Devel/issues

You can reference this commit:
https://lwn.net/Articles/992496/